Day1Main.py

```diff
@@ -0,0 +1,25 @@
+from InputReader import InputReader
+from Day1.DayOnePartOne import FloorCounter
+from Day1.DayOnePartTwo import BasementFinder
+
+input_reader = InputReader("input1.txt")
+raw_input = input_reader.get_raw_input()
+
+floor_counter = FloorCounter(raw_input)
+parsed_input = floor_counter.parse_input()
+floor_counter.count_floors(parsed_input)
+print("The final floor at the end of the input:", floor_counter.get_final_floor())
+
+basement_index_finder = BasementFinder(raw_input)
+basement_index_finder.calculate_basement_entry(parsed_input)
+print("The step at which Santa enters the basement:", basement_index_finder.get_basement_entry())
+
+robs_challenged1 = FloorCounter("(((-2)+6((")
+robs_challenged1_input = robs_challenged1.parse_input()
```

Comment:
```
robs_challenged1 = FloorCounter("(((-2)+6((")
robs_challenged1_input = robs_challenged1.parse_input()
robs_challenged1.count_floors_with_skips(robs_challenged1_input)
print("If Santa is lazy and uses an elevator, and is crazy and hates the number 8, the final floor is:", robs_challenged1.get_final_floor())
```

amikor vegzell a koddal olvasd at ujra h logikus e

fogsz egy floor countert adsz neki random gibrisht mint inputot es elmented ugy h rob_challenged (ezt mar kifejtettem h miert nem jo otlet :D)

utana fogod a rob_challengedet es parsoltatsz vele uj inputot a nev alapjan ez vmi challenge miert parsol???

majd fogod a rob_challenged (ami igazabol floor counter) altal parsolt inputot es megeteted _megint_ a rob_challengeddel

majd elkered a rob_challengetol a final floort 

ezt most garantalom neked h 2 het mulva mar nem fogod tudni h mi a fasz tortenik :D de rajtad kivul senki mas sem tudja mar most

kepzeld el ugy a kodod mintha egy tortenetet meselnel vkinek

letudod kovetni ezt a tortenetet? mennyiszer kerdez vissza? mennyi mellekmagyarazatra lesz szukseged?

a naming horroron kivul 

egyertelmuen a parsingra kell egy masik class
sztem ne mentsd az inputot a ctorban mert limitalod a class ujrahasznalhatosagat

ha valamit el akarsz menteni akkor elmentheted a skippelt floorokat esetleg es amikor ha nem skippelsz akkor ures a lista amikor skippelsz vannak benne ertekek

Reply:
masik, lehet h ujra olvastad de rogton a megiras utan es meg minden a fejedben volt es igy erthetonek tunt

legkozelebb tarts 5-10 perc 1-2 ora szunetet menj el csinalj vmi mast es utana olvasd ujra vagy direkt figyelj arra h hiaba van ott a te fejedben masnak is meg kell ertenie, neked is meg kell ertened akkor is ha nem friss minden

megprobalhatsz amugy rubber duckizni is es akkor lehet feltunobb amikor kimondod a dolgokat h nem passzolnak